src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/builders/PackageSummaryBuilder.java

```diff
@@ -57,6 +62,10 @@
      */
     private final PackageSummaryWriter packageWriter;
 
+    // Maximum number of subpackages and sibling packages to list in related packages table
+    private final static int MAX_SUBPACKAGES = 20;
+    private final static int MAX_SIBLING_PACKAGES = 5;
+
```

Comment:
For future work, move these to a config file.

Short-term "workaround" suggestions still for a separate RFR: annotate these and similar declarations with an internal source-retention annotation of `@Configuration`